src/test/java/net/jqwik/micronaut/JpaSingleTransactionNoSetupPerTryTest.java

```diff
@@ -0,0 +1,41 @@
+package net.jqwik.micronaut;
+
+import io.micronaut.test.annotation.TransactionMode;
+import jakarta.inject.Inject;
+import net.jqwik.api.Property;
+import net.jqwik.api.lifecycle.AfterProperty;
+import net.jqwik.micronaut.annotation.DbProperties;
+import net.jqwik.micronaut.annotation.JqwikMicronautTest;
+import net.jqwik.micronaut.beans.Book;
+
+import javax.persistence.EntityManager;
+import javax.persistence.criteria.CriteriaQuery;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+@JqwikMicronautTest(transactionMode = TransactionMode.SINGLE_TRANSACTION, perTry = true)
+@DbProperties
+class JpaSingleTransactionNoSetupPerTryTest {
+    @Inject
+    private EntityManager entityManager;
+
+    @AfterProperty
```

Comment:
Shouldn't that now be `@AfterTry` to verify that it will be rolled back per try?